designs/2019-allow-all-directives-in-line-comments/README.md

```diff
@@ -0,0 +1,84 @@
+- Start Date: 2019-07-22
+- RFC PR: (leave this empty, to be filled in later)
+- Authors: Jordan Eldredge <jordan@jordaneldredge.com>
+
+# Allow All Directives in Line Comments
+
+## Summary
+
+ESLint currently only allows the following types in directives to be used inside of block comments (`/* Block Comment */`):
+
+* `exported`
+* `global(s)`
+* `eslint-(en|dis)able`
+* `eslint`
+
+Currently if a user adds one of these directives in a line comment (`// Line Comment`) the directive will be silently ignored. Which can be very confusing.
+
+I propose that we allow all directives to work in either type of comment.
+
+## Motivation
+
+As a user of ESLint I occasionally wish to disable a rule, declare a valid global or add some other file level config. In doing so I often forget that these directives are only valid inside block level comments and add them as inline comments by mistake. When the comment fails to have its desired effect I consult the documentation to see what I did wrong. Did I missremember the directive syntax or name? On more than one occasion the problem has been that I used the wrong kind of comment.
```

Comment:
My spellcheck was flagging this one but it also flagged the way I would normally type it, so I came up with a longer way-- disregard if you like.

```suggestion
As a user of ESLint I occasionally wish to disable a rule, declare a valid global or add some other file level config. In doing so I often forget that these directives are only valid inside block level comments and add them as inline comments by mistake. When the comment fails to have its desired effect I consult the documentation to see what I did wrong. Did I remember the directive syntax or name incorrectly? On more than one occasion the problem has been that I used the wrong kind of comment.
```